ui/ozone/platform/wayland/wayland_window.cc

```diff
@@ -367,7 +367,12 @@ void WaylandWindow::Configure(void* data,
 
   window->ResetWindowStates();
   uint32_t* p;
-  wl_array_for_each(p, states) {
+  // wl_array_for_each has a bug in upstream. It tries to assign void* to
+  // uint32_t *, which is not allowed in C++. Explicit cast should be performed.
+  // We can't modify wayland-util.h, because it is fetched with gclient sync.
+  // Thus, use own loop.
+  for (p = (uint32_t*)states->data;
```

Comment:
This built ok for me, because I've modified third_party/wayland/src, a patch of which is impossible to include into pull requests, because wayland/src is fetched during gclient sync. I've just have not noticed that wayland-util.h is not included when pushed the previous patch.

That's an error in wayland upstream.